.idea/runConfigurations/Main.xml

```diff
@@ -1,5 +1,7 @@
 <component name="ProjectRunConfigurationManager">
   <configuration default="false" name="Main" type="Application" factoryName="Application" singleton="false" nameIsGenerated="true">
+    <option name="ALTERNATIVE_JRE_PATH" value="temurin-21" />
+    <option name="ALTERNATIVE_JRE_PATH_ENABLED" value="true" />
```

Comment:
Is this supposed to be here?

Reply:
It just specifies the jre for the run config if someone doesn't have it set